tmt-app/src/Components/Home.js

```diff
@@ -94,9 +103,9 @@ function Home() {
             <Typography variant="CustomHeading2">Take the Quiz</Typography>
           </PrimaryBtn>
 
-          <div className="Home_Button_Main_Link">
+          <FlexContainer sx={{ marginTop: "4.5rem", flexDirection: "column" }}>
             <Link
-              sx={{ color: "Boba.main" }}
+              sx={{ color: "Boba.main", textDecoration: "underline" }}
               underline="none"
               target="_blank"
```

Comment:
I don't think we need to open a new window for this

Reply:
This is for viewing purposes and will be removed after the quiz is merged in